qutebrowser/browser/commands.py

```diff
@@ -976,6 +977,9 @@ def tab_focus(self, index: typing.Union[str, int]=None, count=None):
         if index is None:
             self.tab_next()
             return
+        if index == 0:
+            index = self._count()
+
```

Comment:
Not something directly related to your change, but I think it'd be cleaner to combine this into an `if`/`elif`/`elif` with the check above/below, as they're mutually exclusive - could you please do that?